pkg/obfuscate/sql_tokenizer.go

```diff
@@ -243,7 +267,11 @@ func (tkn *SQLTokenizer) Scan() (TokenKind, []byte) {
 	tkn.SkipBlank()
 
 	switch ch := tkn.lastChar; {
-	case isLeadingLetter(ch):
+	case isLeadingLetter(ch) &&
+		(tkn.cfg.DBMS != DBMSPostgres || ch != '@'):
```

Comment:
```suggestion
		!(tkn.cfg.DBMS == DBMSPostgres && ch == '@'):
```
In my opinion this is easier to read, but maybe that's just me.

Reply:
oh, nice!  Much better!